### PR TITLE
update doc and examples for suspended data

### PR DESCRIPTION
**Pull request by agbanusi**
Updated Examples and Doc

Screenshot 2022-07-05 at 22 34 12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <img height="142" src="https://github.com/GooseFX1/gfx-web-app/blob/dev/public/img/assets/gfx_logo_gradient_lite.svg" />
-  <h3>GooseFX SDK Mono Repo</h3>
+  <h3>GooseFX Typescript SDK</h3>
```
Comment (dovahcrow) [inline README.md:3]: This repo contains not only typescript but also rust sdk

File: README.md [hunk 2]
```diff
@@ -42,27 +39,26 @@ npm install goosefx-ssl-sdk
 ## Get Quotes and price Impact
 
 ```typescript
-import { PublicKey, Connection } from "@solana/web3.js";
-import { Swap } from "goosefx-ssl-sdk";
+import { Connection } from "@solana/web3.js";
+import { Swap } from "goosefx-ssl-sdk;
```
Comment (dovahcrow) [inline README.md:43]: Missing quote?

File: README.md [hunk 3]
```diff
@@ -97,13 +93,38 @@ async function main() {
...
+createTx()
 
 ```
 
-# Build
+## Get the status of the SSL Pool
+
+```typescript
+import { Connection } from "@solana/web3.js";
+import { SSL, ADDRESSES } from "goosefx-ssl-sdk;
```
Comment (dovahcrow) [inline README.md:104]: Missing quote

**Comment by dovahcrow** [comment]
Cool! How about move all the typescript examples from the README in the root to the README in the ts folder?

**Comment by agbanusi** [comment]
> Cool! How about move all the typescript examples from the README in the root to the README in the ts folder?

They are all in the typescript. I added to the root folder because of the npm doc refers to it

File: README.md (hunk 1 above)
Comment (lukewalt) [reply README.md]: Correct, the README at the root of the project says "GooseFX SDK Mono Repo" but this README file is in the `ts` directory so its only referring to the ts sdk.

**Comment by lukewalt** [comment]
All of the links on npm point to the `/ts` directory where the README demos ts code snippets. We can remove the ts examples from the README in the root.

**Review by lukewalt — APPROVED** [review]

**Pull request merged**